app/models/book.rb

```diff
@@ -83,7 +83,7 @@ class Book < ActiveRecord::Base
   has_many :users, through: :purchases
   has_one :failed_update
   has_and_belongs_to_many :authors
-  has_and_belongs_to_many :groups
+  has_and_belongs_to_many :content_buckets
```

Comment:
Inconsistent indentation detected.<br>Prefer `has_many :through` to `has_and_belongs_to_many`.